.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,55 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
+
+jobs:
+    test:
+        name: 'PHP ${{ matrix.php-version }}'
+        runs-on: ubuntu-20.04
+
+        strategy:
+            fail-fast: false
+
+            matrix:
+                php-version:
+                    - 7.3
+                    - 7.4
+                    - 8.0
+                dependencies:
+                    - highest
```

Comment:
i think this should be rewritten to the composer flag --prefer-lowest to be only used on the lowest php version instead.

Reply:
Yeah I think the combiniation between include and normal matrix is buggy in github action I moved all to the include now.